modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
+= Use Studio to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To use Anypoint Studio to configure a connector:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+*  Configure an associated global element for the input source and connector.
+* Create a `.yaml` file or `.properties` file to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+For information about these best practices, see <<introduction/intro-connector-configuration-overview.adoc#best-practices,Best Practices for Configuring Connectors>>.
+
+== Install the Connector
+
+Installing the connector adds it to a Studio project. Studio provides two ways to install a connector:
+
+* Use the Exchange application in the Studio taskbar.
+
+* Use the Exchange icon in the Mule Palette view.
+
+(*Reviewers, is there any advantage to using one of these methods for installing a connector* )
+
+=== Install the Connector from the Studio Taskbar
+
+. In Studio, create a Mule project.
+. Click the Exchange icon *(X)* in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for the connector, for example, *Amazon SQS*.
+. Select the connector.
+. Click *Add to project*.
+. Follow the prompts to install the connector.
+
+=== Install the Connector from the Mule Palette View
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, enter the connector name in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source for the Connector
```

Comment:
Same as my comment above. You are configuring an input source for the flow, not the connector.